README.md

```diff
@@ -389,6 +389,20 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
 
 After deploying the DBP, when accesing one of the applications, for example Process Services, if you receive the error message *We're sorry Invalid parameter: redirect_uri*, the `redirectUris` parameter provided for deployment is invalid. Make sure the `alfresco-infrastructure.alfresco-identity-service.client.alfresco.redirectUris` parameter has a valid value when installing the chart. For more details on how to configure it, check this [guide](https://github.com/Alfresco/alfresco-identity-service#changing-alfresco-client-redirecturis).
 
+**No Digital Business Platform components can be accessed**
+
+Please make sure that you are not running any server locally that would occupy port 80. For example macOs Server runs specifically on this port, please disable it before deploying the Digital Business Platform.
```

Comment:
macOs->macOS

Reply:
I think explaining some of the troubleshooting commands would be useful to such as -

kubectl delete pvc and pv when cleaning up

Reply:
Jesson, so listing the commands 

> kubectl get pvc -n $DESIREDNAMESPACE
> kubectl delete pvc oldpvc -n $DESIREDNAMESPACE
are not sufficient?